lib/sonata/tip/getChannelTipInfo.ts

```diff
@@ -0,0 +1,31 @@
+import pregenerateChannelWallet from "../../privy/pregenerateChannelWallet";
+import searchChannels from "../../privy/searchChannels";
+
+const FEE = .1;
+
+function getChannelId(url: string) {
+  const match = url.match(/^(\w+):\/\/([^/]+)\/channel\/(\w+)$/);
+  if (match) return match[3];
+  return null;
+}
```

Comment:
each file should do `exactly` one thing
- actual: this file has `2 functions` declared
- expected: this file has only `one` function declared
- could you please move `getChannelId` to a standalone lib?